test/functional/test_framework/test_framework.py

```diff
@@ -283,7 +284,11 @@ def assert_start_raises_init_error(self, i, extra_args=None, expected_msg=None,
                 if expected_msg is not None:
                     log_stderr.seek(0)
                     stderr = log_stderr.read().decode('utf-8')
-                    if expected_msg not in stderr:
+                    if use_re:
+                        match = re.search(expected_msg, stderr)
```

Comment:
nit nit, always treat as regexp and remove option?

Reply:
no, then I'd have to change all the tests